support-data-rest/src/main/java/uk/ac/ebi/uniprot/api/disease/response/converter/DiseaseJsonMessageConverter.java

```diff
@@ -0,0 +1,29 @@
+package uk.ac.ebi.uniprot.api.disease.response.converter;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import uk.ac.ebi.uniprot.api.rest.output.converter.AbstractJsonMessageConverter;
+import uk.ac.ebi.uniprot.cv.disease.Disease;
+import uk.ac.ebi.uniprot.json.parser.disease.DiseaseJsonConfig;
+import uk.ac.ebi.uniprot.search.field.DiseaseField;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Arrays;
+
+public class DiseaseJsonMessageConverter extends AbstractJsonMessageConverter<Disease> {
```

Comment:
I think this class is no longer required. once you make the AbstractJsonMessageConverter not abstract anymore